### ISSUE TITLE
Devices (maybe only Osram+) never go to unavailable state, when they have no power

### ISSUE THREAD

**Issue by mmichalko** [zigpy/zigpy-deconz#57]
Hi,

I am a new owner of Raspbee running on Raspberry Pi 3 B+. I bought few Osram Smart+ tunable white bulbs, which are connected via ZHA to Hass.io. Everything works nice, I can turn a bulb on/off, dimm or change light temperature. 

Problem begins, when I switch the wall switch off, which cuts power to the bulb. HA never changes the state of the bulb to unavailable (like with my yeelight wlan bulbs). The bulb keeps the previous state in HA (when it was off is off, when it was on is on) and when I click the switch in UI, it changes for few seconds and then the switch returns itself to previous position.

What I expect to happen with the bulb: 
it should go to unavailable state like Yeelight wlan bulbs.

Bottom Line:
I have played with Node Red in Hass.io and when I debug the bulb with "poll: state" node, I see no change in state, after I cut the power to the bulb - I still keep getting new polled events with the bulb state "on", even if it has no power.

**Comment by Adminiuga** [comment]
Not a zigpy-deconz issue. ZHA component of HA marks entities unavailable only if no traffic is received within 2 hour windows

**Comment by mmichalko** [comment]
Ok, thanks for the quick reply. Just quick question - this means if I switch to deconz plugin, I wouldn't use ZHA and there it could potentially work?

Also, I am correct, that this ZHA issue should be reported directly to HA github (https://github.com/home-assistant/home-assistant/issues)?

**Comment by Adminiuga** [comment]
Haven't tried deconz plugin, so I don't really know how it would behave. I would not expect an immediate "powered off" device detection, as zha devices usually don't send any notification about being powered down. 

For ZHA issue, yes, open issue with home-assistant github issues and tag it as ZHA integration.

**Comment by mmichalko** [comment]
Ok, thanks!
